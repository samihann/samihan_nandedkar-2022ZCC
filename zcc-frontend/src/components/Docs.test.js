import React from "react"
import ReactDOM from "react-dom"
import Docs from './Docs'
import Enzyme, { mount, shallow, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('Docs', () => {
    it('should render correctly', () => {
        const component = shallow(<Docs />);
      
        expect(component).toMatchSnapshot();
      });
 });