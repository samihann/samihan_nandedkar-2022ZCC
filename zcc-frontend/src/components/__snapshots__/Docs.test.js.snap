// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Docs should render correctly 1`] = `ShallowWrapper {}`;

exports[`Navbar should render correctly 1`] = `ShallowWrapper {}`;
